src/Tribe/Views/V2/View.php

```diff
@@ -1679,7 +1699,8 @@ static function () {
 			'request_date'         => Dates::build_date_object( $this->context->get( 'event_date', $today ) ),
 			'rest_url'             => $endpoint->get_url(),
 			'rest_method'          => $endpoint->get_method(),
-			'rest_nonce'           => $rest_nonce,
+			'rest_nonce_a'           => $rest_nonce_a,
```

Comment:
One option here is to instead serialize the nonces on the existing field and not require as much of a lift. But I think having explicitly two nonces is a bit clearer on what we are doing and follow the logic thread, versus a serialized object.

Reply:
I vote for the two nonces showing. I think it makes is clearer too.